src/pretix/base/models/items.py

```diff
@@ -764,6 +764,8 @@ class ItemVariation(models.Model):
     :type default_price: decimal.Decimal
     :param original_price: The item's "original" price. Will not be used for any calculations, will just be shown.
     :type original_price: decimal.Decimal
+    :param require_approval: If set to ``True``, orders containing this product can only be processed and paid after approved by an administrator
```

Comment:
Maybe make it consistent regarding variation here as well?
```suggestion
    :param require_approval: If set to ``True``, orders containing this product's variation can only be processed and paid after approved by an administrator
```